src/libYARP_dev/src/Drivers.cpp

```diff
@@ -87,7 +87,10 @@ class DriversHelper : public YarpPluginSelector {
             settings.readFromSearchable(prop,name);
             settings.open(lib);
             ConstString location = lib.getName().c_str();
-            if (location=="") continue;
+            if (location=="") {
```

Comment:
Is location set empty after the library failed loading? Or does this work only for empty location? Do you mind adding a comment here to explain?